self-hosted-login/okta-aspnet-mvc-example/Startup.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using System.Configuration;
+using Microsoft.Owin;
+using Microsoft.Owin.Security;
+using Microsoft.Owin.Security.Cookies;
+using Okta.AspNet;
+using Owin;
+
+[assembly: OwinStartup(typeof(okta_aspnet_mvc_example.Startup))]
+
+namespace okta_aspnet_mvc_example
+{
+    public class Startup
+    {
+        public void Configuration(IAppBuilder app)
+        {
+            app.SetDefaultSignInAsAuthenticationType(CookieAuthenticationDefaults.AuthenticationType);
+
+            app.UseCookieAuthentication(new CookieAuthenticationOptions());
+
+            app.UseOktaMvc(new OktaMvcOptions()
+            {
+                OktaDomain = ConfigurationManager.AppSettings["okta:OktaDomain"],
+                ClientId = ConfigurationManager.AppSettings["okta:ClientId"],
+                ClientSecret = ConfigurationManager.AppSettings["okta:ClientSecret"],
+                RedirectUri = ConfigurationManager.AppSettings["okta:RedirectUri"],
+                PostLogoutRedirectUri = ConfigurationManager.AppSettings["okta:PostLogoutRedirectUri"],
+                GetClaimsFromUserInfoEndpoint = true,
```

Comment:
Are the defaults okay for GetClaims... and Scope?

Reply:
Defaults:
```
public bool GetClaimsFromUserInfoEndpoint { get; set; } = false;
public static readonly string[] Scope = new string[] { "openid", "profile" };
```